### PR TITLE
🧹 update bubbletea rendering go dependencies

### PR DESCRIPTION
**Pull request by chris-rock**
superseeds #1106

### CODE REVIEW

File: cli/components/list.go [hunk 1]
```diff
@@ -39,8 +39,9 @@ func (d itemDelegate) Render(w io.Writer, m list.Model, index int, listItem list
 
 	fn := itemStyle.Render
 	if index == m.Index() {
-		fn = func(s string) string {
-			return selectedItemStyle.Render("> " + s)
+		fn = func(strs ...string) string {
+			strs = append([]string{"> "}, strs...)
```
Comment (czunker) [inline cli/components/list.go]: The renderer changed. It now looks like this:
image

The selected item is now intended one space. When we want to keep it as it was before, we would need to remove the space in this line.

Comment (chris-rock) [reply cli/components/list.go]: We should remove that and keep the current behavior.

Comment (chris-rock) [reply cli/components/list.go]: Thank you @czunker I updated and fixed this issue

**Review by czunker — APPROVED** [review]
LGTM. Thanks @chris-rock

**Pull request merged**